tests/112-test-use-devs-to-create-container-root-volume.sh

```diff
@@ -27,12 +27,12 @@ CONTAINER_ROOT_LV_MOUNT_PATH=$root_lv_mount_path
 CONTAINER_THINPOOL=container-thinpool
 EOF
 
- # Run docker-storage-setup
- $DSSBIN $infile $outfile >> $LOGS 2>&1
+ # Run container-storage-setup
+ $CSSBIN $infile $outfile >> $LOGS 2>&1
 
  # Test failed.
  if [ $? -ne 0 ]; then
-    echo "ERROR: $testname: $DSSBIN failed." >> $LOGS
```

Comment:
Sure, if that is what you want.  I still think we should change all files on disk in one patch and then change internal names as a separate patches.